Bootstrap dependency conflict

Error when installing project 'npm install'.
NG Bootstrap needs to be updated.

npm error code ERESOLVE
npm error ERESOLVE could not resolve
npm error
npm error While resolving: @ng-bootstrap/ng-bootstrap@16.0.0
npm error Found: @angular/common@18.0.5
npm error node_modules/@angular/common
npm error   @angular/common@"^18.0.2" from the root project
npm error   peer @angular/common@"^18.0.0 || ^19.0.0" from @angular/cdk@18.0.5
npm error   node_modules/@angular/cdk
npm error     @angular/cdk@"^18.0.2" from the root project
npm error   7 more (@angular/forms, @angular/platform-browser, ...)
npm error
npm error Could not resolve dependency:
npm error peer @angular/common@"^17.0.0" from @ng-bootstrap/ng-bootstrap@16.0.0
npm error node_modules/@ng-bootstrap/ng-bootstrap
npm error   @ng-bootstrap/ng-bootstrap@"^16.0.0" from the root project
npm error
npm error Conflicting peer dependency: @angular/common@17.3.12
npm error node_modules/@angular/common
npm error   peer @angular/common@"^17.0.0" from @ng-bootstrap/ng-bootstrap@16.0.0
npm error   node_modules/@ng-bootstrap/ng-bootstrap
npm error     @ng-bootstrap/ng-bootstrap@"^16.0.0" from the root project
npm error
npm error Fix the upstream dependency conflict, or retry
npm error this command with --force or --legacy-peer-deps
npm error to accept an incorrect (and potentially broken) dependency resolution.

--- a/package.json
+++ b/package.json
@@ -37,7 +37,7 @@
     "@codemirror/theme-one-dark": "^6.1.2",
     "@codemirror/view": "^6.14.1",
     "@compodoc/compodoc": "^1.1.23",
-    "@ng-bootstrap/ng-bootstrap": "^16.0.0",
+    "@ng-bootstrap/ng-bootstrap": "^17.0.1",
     "@ngx-translate/core": "^15.0.0",
     "@ngx-translate/http-loader": "^8.0.0",
     "@popperjs/core": "^2.11.6",

